### ISSUE TITLE
No/low disk space warning.

### ISSUE THREAD

**Issue by barisyild** [getsentry/self-hosted#3008]
### Problem Statement

When disk space becomes full or low, sentry starts to experience various problems, for example "envelope buffer capacity exceeded" occurs because sentry cannot process new issues.

### Solution Brainstorm

A warning can be added where we can see the disk space or indicate that disk space is running low/full.

### Product Area

Other

**Comment by hubertdeng123** [comment]
It is unlikely we'll implement this feature, but something that may be helpful for you to do is set up some sort of cron monitor to ensure your disk space doesn't get too low. Or, you could set up some sort of external integration that can check the disk space on your cloud instance if you're using GCP or AWS.

**Comment by brunis** [comment]
You could also change the minimum requirements to the current recommended requirements.. and then double all the numbers in the Recommended..  for every release the demands keep going up .. I have a VM with recommended specs, I had to add 20GB of swap to avoid sentry crashing my VM on startup..
It seems bizarre my logging application (sentry) needs 2-3x the hardware of the application logging to it..

**Comment by hubertdeng123** [comment]
Yes, we could implement this feature. However, it is not a priority at the moment.

**Comment by aldy505** [comment]
Closing this since there are no progress and the solution would be for you to manage your own storage freespace rather than relying on Sentry's.
